doc/Makefile

```diff
@@ -199,7 +199,7 @@ all: hpp_tutorial.install hpp-gepetto-viewer.install
 test-ci: baxter_common.install romeo.install universal_robot.install \
 	hpp-environments.install
 	${MAKE} hpp_tutorial.install hpp-gepetto-viewer.install \
-	hpp_universal_robot.install; \
+	hpp_universal_robot.install && \
```

Comment:
Does this make *make* behave differently ?

Reply:
Yes, if the first command fails, the second is executed anyway, it append a few times to me before I could understand the issue…